app/models/post.rb

```diff
@@ -2,4 +2,14 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   has_many :comments
   has_many :likes
+
+  after_create :update_post_counter
+
+  def update_post_counter
+    user.increment!(:posts_counter)
+  end
```

Comment:
- [ ] You can make the method private here.

Reply:
Solved :+1: